src/main/java/ch/njol/skript/aliases/Aliases.java

```diff
@@ -56,25 +31,62 @@
 import java.util.Map;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.lang.script.Script;
 
 public abstract class Aliases {
-
 	static final boolean USING_ITEM_COMPONENTS = Skript.isRunningMinecraft(1, 20, 5);
 
 	private static final AliasesProvider provider = createProvider(10000, null);
 	private static final AliasesParser parser = createParser(provider);
-	
+	// this is not an alias!
+	private final static ItemType everything = new ItemType();
+	private final static Message m_empty_string = new Message("aliases.empty string");
+	private final static ArgsMessage m_invalid_item_type = new ArgsMessage("aliases.invalid item type");
+	private final static Message m_outside_section = new Message("aliases.outside section");
+	private final static RegexMessage p_any = new RegexMessage("aliases.any", "", " (.+)", Pattern.CASE_INSENSITIVE);
+	private final static RegexMessage p_every = new RegexMessage("aliases.every", "", " (.+)", Pattern.CASE_INSENSITIVE);
+	private final static RegexMessage p_of_every = new RegexMessage("aliases.of every", "(\\d+) ", " (.+)", Pattern.CASE_INSENSITIVE);
+	private final static RegexMessage p_of = new RegexMessage("aliases.of", "(\\d+) (?:", " )?(.+)", Pattern.CASE_INSENSITIVE);
+	/**
+	 * Go through these whenever aliases are reloaded, and update them.
+	 */
+	private static final Map<String, ItemType> trackedTypes = new HashMap<>();
+	/**
+	 * If user had an obscure config option set, don't crash due to missing
+	 * Java item types.
+	 */
```

Comment:
```suggestion
	 */
	
```

Reply:
i don't think this space is where it should be

Reply:
lol oops

Reply:
So...Should I add it?

Reply:
no